src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="calendar page__calendar calendar--month-length--31 calendar--first-day--sun">
```

Comment:
make calendar--month-length--29, better)

Reply:
Great)
My falt, sorry)

Reply:
I used current values of month (Feb 2023, fd: Wednesday, length: 28 days per month) ✔✔✔